psydac/core/bsplines.py

```diff
@@ -504,6 +516,7 @@ def elements_spans(knots, degree, out=None):
        spans  = np.searchsorted( knots, breaks[:-1], side='right' ) - 1
 
     """
+    knots = np.asarray(knots)
```

Comment:
As above

Reply:
Done in https://github.com/pyccel/psydac/commit/54fa1fb4c2afba79c77392308a193bad59161fee